src/components/Leaderboard.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { FirebaseClassContext } from './App';
+import './styles/Leaderboard.scss';
+
+interface Score {
+  name?: string;
+  email?: string;
+  level?: string;
+  score?: number;
+}
```

Comment:
How is this different than `UserInfo`

Reply:
Oops good catch, meant to remove email and level